examples/BresserWeatherSensorBasic/BresserWeatherSensorBasic.ino

```diff
@@ -62,6 +62,8 @@ void setup() {
     Serial.begin(115200);
     Serial.setDebugOutput(true);
 
+    Serial.printf("Starting execution...\n");
```

Comment:
I think this could be dicarded.

Reply:
Yes, I think this can be left for my own scripts... :-)